server/src/main/java/org/candlepin/model/AsyncJobStatus.java

```diff
@@ -215,25 +207,29 @@ public JobDataJsonConverter() {
     @Column(name = "end_time")
     private Date endTime;
 
-    @Column(name = "job_data")
-    @Convert(converter = JobDataJsonConverter.class)
-    private SerializedJobData jobData;
+    @ElementCollection(fetch = FetchType.LAZY)
+    @CollectionTable(name = "cp_async_job_arguments", joinColumns = @JoinColumn(name = "job_id"))
+    @MapKeyColumn(name = "name", nullable = false)
+    @Column(name = "value")
+    private Map<String, String> arguments;
+
+    @Column(name = "job_result")
+    private String result;
```

Comment:
Haven't gone back to check. The first method ended up causing tons of problems and not working, so I just dropped it to a standard field for the interim. I think it'll be fine since we will ever have around 10 populated instances per node at any given time outside of the remaining problem area that needs to be addressed in the bulk-fetching of jobs.